src/AnimatedGIF.ts

```diff
@@ -159,7 +170,7 @@ class AnimatedGIF extends Sprite
      * @param options - Options to use.
      * @returns
      */
-    static fromBuffer(buffer: ArrayBuffer, options?: Partial<AnimatedGIFOptions>): AnimatedGIF
+    static fromBuffer(buffer: ArrayBuffer, options?: Partial<DefaultOptions>): AnimatedGIF
```

Comment:
Another way to do this without creating another interface is to use `Omit`. Would be my preference.

```suggestion
    static fromBuffer(buffer: ArrayBuffer, options?: Partial<Omit<AnimatedGIFOptions, 'width'|'height'>>): AnimatedGIF
```